### DISCUSSION TITLE
Free for All mode (like Smash Bros, WWE Video Games)

### DISCUSSION THREAD

**Discussion by ghost** [ikemen-engine/Ikemen-GO#1283] [Feature Request]
### Is your feature request related to a problem? Please describe.

It is not, its just an idea!

### Describe the solution you'd like

In this mode, each character is assigned their own "teamside", and will trade blows with one another. The match will continue forward until one player is standing, when that happens the last one standing will win the match.

Double KO conditions are when all players are knocked out, and none survive the match, resulting in a draw game.

### Additional context

https://www.youtube.com/watch?v=kdTem75n4pI
Here's an example of how it would basically work.

**Comment by Gorilli09** [comment]
That would not be likely going to happen.
